core/Updates/1.8.4-b1.php

```diff
@@ -47,7 +47,7 @@ public function getMigrations(Updater $updater)
 
             // remove protocol and www and store information in url_prefix
             $this->migration->db->sql(
-            "   UPDATE `$action`
+                "   UPDATE `$action`
```

Comment:
Should we remove the leading spaces in the query? Looks like it was only added for visual code formatting reasons rather than needed/functional. What do you think?

Reply:
I would remain with the changes that are needed for CS and were done using phpcbf. For this case it might be fine to change as update scripts are not tested, for other cases such a change could cause test failures...